src/components/media/controls/controls.tsx

```diff
@@ -0,0 +1,274 @@
+import './controls.css'
+
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Icon from '@mui/material/Icon'
+import IconButton from '@mui/material/IconButton'
+import LinearProgress from '@mui/material/LinearProgress'
+import Slider from '@mui/material/Slider'
+import Typography from '@mui/material/Typography'
+import React, { useState } from 'react'
+
+import { formatDurationTime } from '../../helper'
+
+interface MediaIconButtonProps {
+  onClick: () => void
+  action: 'play' | 'pause' | 'forward' | 'replay' | 'volumeUp' | 'volumeOff'
+  className?: string
+}
+
+export function MediaIconButton(props: MediaIconButtonProps) {
+  const controls = {
+    play: { symbol: 'play_circle', label: 'play' },
+    pause: { symbol: 'pause_circle', label: 'pause' },
+    forward: { symbol: 'forward_10', label: 'forward ten seconds' },
+    replay: { symbol: 'replay_10', label: 'replay ten seconds' },
+    volumeUp: { symbol: 'volume_up', label: 'mute' },
+    volumeOff: { symbol: 'volume_off', label: 'unmute' },
+  }
+  return (
+    <IconButton
+      onClick={props.onClick}
+      aria-label={`click to ${controls[props.action].label}`}
+      className={props.className}
+      data-cy={`${props.action}-button`}
+    >
+      <Icon color="primary">
+        <span className="material-symbols-rounded">
+          {controls[props.action].symbol}
+        </span>
+      </Icon>
+    </IconButton>
+  )
+}
+
+interface MediaControls {
```

Comment:
I am wondering if we should put all the interfaces at the top of the file to make them easier to manage. There's other interfaces being defined in the middle of the file.